testdata/ia/shipped/rules.yml

```diff
@@ -0,0 +1,28 @@
+---
+rules:
+    - name: shipped_rules
+      version: 1
+      help: MySQL connections in use
```

Comment:
@palash25 @artemgavrilov May you please work together to rename "help" to "summary" there, in platform repo, Check Service, etc?

Reply:
will do

Reply:
have drafted PRs for both platform and checked repos, will request review once the CI is green